test/benchmark/lmbench/ext2_create_delete_files_0k_ops/run.sh

```diff
@@ -6,4 +6,4 @@ set -e
 
 echo "*** Running the LMbench file system create/delete test (Ext2) ***"
 
-/benchmark/bin/lmbench/lat_fs -s 0k -P 1 /ext2
\ No newline at end of file
+/benchmark/bin/lmbench/lat_fs -s 0k -P 1 -W 30 -N 231 /ext2
```

Comment:
This number of `231` is too magical... People will wonder why this particular number. Would a number of `200` or `300` make any differences?

Reply:
I tested 200 and 300; some benchmarks were changed to 200, others to 300.